### PR TITLE
update ci script to remediate and verify

### PR DESCRIPTION
**Pull request by ppadmavilasom**
do remediation scripts, then verify and use this as the criteria for ci pass
update ci status in README

### CODE REVIEW

File: examples/Dockerfile [hunk 1]
```diff
@@ -16,4 +16,7 @@ install python3 \
 shadow \
 photon-release -qy
 
+COPY remediate_and_verify.sh /
```
Comment (sidcha) [inline examples/Dockerfile]: This is not needed. remediate_and_verify.sh is already available in /src/example/ and cannot work without the volume being mounted with -v option.

File: examples/remediate_and_verify.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+pushd /src
+ansible-playbook  -i examples/chroots cis.playbook.yml --tags "cis" \
+&& \
+ansible-playbook  -i examples/chroots cis.playbook.yml --tags "cis" -e verify=true
+popd
```
Comment (sidcha) [inline examples/remediate_and_verify.sh:6]: Can we void pushd/popd with absolute paths?

**Review by satya-rajesh — APPROVED** [review]

File: examples/remediate_and_verify.sh (hunk 2 above)
Comment (ppadmavilasom) [reply examples/remediate_and_verify.sh:6]: do you see any issues with it in particular?

Comment (sidcha) [reply examples/remediate_and_verify.sh:6]: No. For just a couple of lines felt it was a bit excessive. If more work is going to be done here, then it makes sense.

**Review by sidcha — APPROVED** [review]

**Pull request merged**